tox.ini

```diff
@@ -73,6 +73,7 @@ deps =
     numpy125: numpy==1.25.*
     numpy20: numpy==2.0.*
     numpy21: numpy==2.1.*
```

Comment:
should we still be testing with numpy 1.20?

Reply:
oh whoops wrong line, pretend this is for the `120` line above

Reply:
doesn't hurt